server.js

```diff
@@ -0,0 +1,11 @@
+import express from "express";
+import userRouter from "../ds-02-posting-like-hell/router/expressRouter.js";
```

Comment:
Actually this import could be shortened to:
`import userRouter from './router/expressRouter.js'`